packages/widget/src/components/common/buttons/button.ts

```diff
@@ -0,0 +1,55 @@
+import type { HTMLTemplateResult } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+import { html } from 'lit';
+import { classMap } from 'lit/directives/class-map.js';
+import { when } from 'lit/directives/when.js';
+import { ifDefined } from 'lit/directives/if-defined.js';
+import { BaseComponent } from '../base-component/base-component';
+import { loaderIcon } from '../../../assets';
+import { buttonStyles } from './button.styles';
+
+@customElement('sygma-button')
+export class Button extends BaseComponent {
+  static styles = buttonStyles;
+
+  @property({})
+  text: string = '';
+
+  @property({ type: Object })
+  onClick: () => void = () => {};
+
+  @property({ type: Boolean, reflect: true })
```

Comment:
I wonder why the reflect true? 🤔

Reply:
I thought about styling the attribute similar to https://lit.dev/docs/components/properties/#:~:text=Permalink%20to%20%E2%80%9CEnabling,%23

But used the class in the end. So we can remove it to not overcomplicate the code